src/sidebar/components/Sidebar/SidebarContainer/index.tsx

```
@@ -176,14 +176,7 @@ const Sidebar = ({
                     }
                     textValue={conversation.title}
                   >
-                    <Markdown
-                      components={{
-                        p: ({ children }) => <span>{children}</span>,
-                        strong: ({ children }) => <span>{children}</span>
-                      }}
-                    >
-                      {conversation.title}
-                    </Markdown>
+                    <Markdown className="[&>p]:truncate" children={conversation.title} />
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)